modules/hal_nordic/nrfs/backends/nrfs_backend_ipc_service.c

```diff
@@ -172,6 +172,8 @@ nrfs_err_t nrfs_backend_send(void *message, size_t size)
 
 nrfs_err_t nrfs_backend_send_ex(void *message, size_t size, k_timeout_t timeout, bool high_prio)
 {
+	static struct ipc_data_packet tx_data;
```

Comment:
Does it need to be `static`, actually? It's used only for putting the data packet into the queue. Actually it could be placed inside that `if (size <= MAX_PACKET_DATA_SIZE)` below.
Similar story with `rx_data`.

Reply:
changed to non static